src/tests/ftest/nvme/pool_exclude.py

```diff
@@ -9,8 +9,7 @@
 import threading
 import re
 
-from test_utils_pool import TestPool
-from test_utils_base import LabelGenerator
+from test_utils_pool import add_pool
```

Comment:
(pylint-syntax-error) Cannot import 'test_utils_pool' due to syntax error 'invalid syntax (<unknown>, line 53)'